VTEX - Marketplace APIs.json

```diff
@@ -2384,6 +2608,147 @@
+                    },
+                    "useSellerPaymentMethod": {
+                        "description": "If the store uses the seller payment method (`true`) or not (`false`).",
+                        "type": "boolean"
+                    },
+                    "name": {
+                        "description": "Affiliate name.",
+                        "type": "string"
+                    },
+                    "salesChannel": {
+                        "description": "Seller's [affiliate](https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187) that will be associated with the marketplace's sales channel. Through the affiliate, the seller determines which of his sales channel will be used for the mapping.",
+                        "type": "string"
+                    },
+                    "searchURIEndpoint": {
+                        "description": "Search URI endpoint.",
+                        "type":  "string",
+                        "nullable": true
+                    },
+                    "searchURIEndpointVersion": {
+                        "description": "Search URI endpoint version.",
+                        "type": "string"
+                    },
+                    "searchURIEndpointAvailableVersions": {
+                        "description": "Search URI endpoint available versions.",
+                        "type": "array",
+                        "items": {
+                            "description": "Search URI endpoint available version.",
+                            "type": "string"
+                        }
+                    },
+                    "minimumValueAccumulated": {
+                        "description": "Minimum value accumulated in the cart to checkout the order.",
+                        "type": "integer",
+                        "nullable": true
+                    }
+                }
+            },
+            "requestUpdateAffiliate":{
+                "type": "object",
+                "properties": {
+                    "followUpEmail": {
+                        "description": "Email of the store employee responsible for managing the order.",
+                        "type": "string"
+                    },
+                    "useSellerPaymentMethod": {
+                        "description": "If the store uses the seller payment method (`true`) or not (`false`).",
+                        "type": "boolean"
+                    },
+                    "name": {
+                        "description": "Affiliate name.",
+                        "type": "string"
+                    },
+                    "salesChannel": {
+                        "description": "Seller's [affiliate](https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187) that will be associated with the marketplace's sales channel. Through the affiliate, the seller determines which of his sales channel will be used for the mapping.",
```

Comment:
I found this description confusing and it doesn't sound right. If there are fields above already for the affiliate ID and name, why would this field indicate the affiliate. Shouldn't this be the ID of the sales channel ID that is associated with the affiliate?